crates/miden-lib/src/account/auth/rpo_falcon_512_multisig.rs

```diff
@@ -9,6 +9,26 @@ use crate::account::components::rpo_falcon_512_multisig_library;
 // MULTISIG AUTHENTICATION COMPONENT
 // ================================================================================================
 
+/// Configuration for [`AuthRpoFalcon512MultisigConfig`] component.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AuthRpoFalcon512MultisigConfig {
+    /// List of procedures with authentication thresholds different than the default.
+    pub proc_threshold_map: Vec<(Word, u32)>,
+}
```

Comment:
nit: shouldn't:
```rust
/// Configuration for [`AuthRpoFalcon512MultisigConfig`] component.
```
be
```rust
/// Configuration for [`AuthRpoFalcon512Multisig`] component.
```

Also, is there a reason not to put all multisig config settings here? i.e., make it look something like:
```rust
pub struct AuthRpoFalcon512MultisigConfig {
    pub approvers: Vec<PublicKeyCommitment>,
    pub default_threshold: u32,
    pub proc_threshold_map: Vec<(Word, u32)>,
}
```

And if we make the fields in this struct private, we could guarantee internal consistency after construction.